docs/api-docs/analytics/bodl-overview.md

```diff
@@ -0,0 +1,427 @@
+    return response.json();
+  })
+  .then(function(orderJson) {
+    var orderQty = 0;
+    var lineItems = [];
+
+    for (i = 0; i < orderJson.lineItems.physicalItems.length; i++) {
+      var thisItem = orderJson.lineItems.physicalItems[i];
+      orderQty += thisItem.quantity;
+      lineItems.push({
+        "content_id": thisItem.productId,
+        "content_name": thisItem.name,
+        "currency": orderJson.currency.code,
+        "price": thisItem.salePrice,
+        "value": thisItem.extendedSalePrice,
+        "quantity": thisItem.quantity,
+        "content_type": "product_group"
+      });
+    }
+
+    for (i = 0; i < orderJson.lineItems.digitalItems.length; i++) {
+      var thisItem = orderJson.lineItems.digitalItems[i];
+      orderQty += thisItem.quantity;
+      lineItems.push({
+        "content_id": thisItem.productId,
+        "content_name": thisItem.name,
+        "currency": orderJson.currency.code,
+        "price": thisItem.salePrice,
+        "value": thisItem.extendedSalePrice,
+        "quantity": thisItem.quantity,
+        "content_type": "product_group"
+      });
+    }
+
+    for (i = 0; i < orderJson.lineItems.giftCertificates.length; i++) {
+      var thisItem = orderJson.lineItems.giftCertificates[i];
+      orderQty += thisItem.quantity;
+      lineItems.push({
+        "content_id": thisItem.type,
+        "content_name": thisItem.name,
+        "currency": orderJson.currency.code,
+        "price": thisItem.amount,
+        "value": thisItem.amount,
+        "quantity": thisItem.quantity,
+        "content_type": "product_group"
+      });
+    }
+
+    sampleAnalyticsProvider.instance('<%= property_id %>').track('Purchase', {
+        "contents": lineItems,
+        "value": orderJson.orderAmount,
+        "quantity": orderQty,
+        "currency": orderJson.currency.code
+    });
+  });
+</script>
+```
+
+## Registration
+The following snippet executes when a shopper successfully creates a new account. It initiates capture of any data the sample analytics provider collects about account registration.
+
+```handlebars title="Sample Script Code Start: Registration" lineNumbers
+<script>
+  if (window.location.pathname.indexOf('/login.php') === 0 && BODL.getQueryParamValue('action') === 'account_created') {
+    sampleAnalyticsProvider.instance('<%= property_id %>').track('Registration');
+  }
+</script>
+```
+
+## Search
+
+A search snippet can capture data about shoppers' searches for products. Our current working implementation contains distortions that limit data capture to only the first category ID in a search. For this reason, we have not provided a search example snippet.
+
+<!-- Please take note that there is a built-in tracker for the category. However, we have commented out the tracker due to known distortions, which cause reporting of only the first category ID. For this reason, we have not provided a search example snippet. Only reactivate if needed; use it at your own risk. -->
```

Comment:
@bc-derrekpearson & @becomevocal -- Please think about what you want to say to an external audience here. The commented text is yours and is not appropriate for publication in external docs because it speaks to specific properties of something that is not here.